src/frontend/src/btc/components/send/BtcSendReview.svelte

```diff
@@ -18,4 +29,12 @@
 		}) || invalidAmount(amount);
 </script>
 
-<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={invalid} />
+<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={disableSend}>
+	<svelte:fragment slot="info">
```

Comment:
Maybe we can actually extract this in a separate component? Like BtcSendHasPendingTransactions or something? That would scope the UI feature, WDYT?

Reply:
Good idea, done.